src/pages/v2/Root.tsx

```diff
@@ -14,8 +15,16 @@ const Wrapper = styled.div`
   height: 100%;
 `
 
+const MainWrapper = styled.div`
+  width: 100;
+  height: 100vh;
+`
+
 const Root = () => {
+  const currentLocation = useLocation()
+  const isHistoryPage = currentLocation.pathname.includes('/history')
   const { chainId, library } = useActiveWeb3React()
+  const [status, setStatus] = useState(1)
```

Comment:
@raynear this status property should be set to the status of the last transaction. 0 is success, 1 is pending, 2 is failed. I think it is related to data base, that is why I tagged you.